### ISSUE TITLE
Unable to find secure ID

### ISSUE THREAD

**Issue by VigneshRamanathan101** [sopov/resumeio2pdf#28]
This is related to issue #27 

After logging in I did a https://resume.io/api/app/resumes from browser and my response had "secureId": null,

**Comment by shashwatmathur** [comment]
Yes, they have blocked the secure id that was supposed to be a private URL

**Comment by saiakarsh193** [comment]
so is there no way to download the PDF for free anymore?

**Comment by ibx99** [comment]
@saiakarsh193 No, there is, but only if have an old account on resumeio. See the thing is they have stopped exposing the secureID for the new accounts, but the old accounts still have them exposed. At least for now...

**Comment by patrickkdev** [comment]
My account is old. I have used this before. Now you can't share the link anymore :(

**Comment by patrickkdev** [comment]
Guys I just found an awesome way to get the secureId. Make sure you're logged into resume.io, then access the api endpoint:

https://resume.io/api/app/resumes

There you will find field 'secureId' : )

**Comment by faustinozanetto** [comment]
I've managed to get it by doing this: 
![image](https://github.com/sopov/resumeio2pdf/assets/11697697/14d07265-560e-4fcf-be83-fbf9e0e3584e)
I do have an old account like @ibx99 said so I don't really know if it will work on newer ones.

**Comment by rubenheymans** [comment]
same for me I have one newer cv, in English, and an older one in Dutch. The one in Dutch has a secureId.
See info here:
https://resume.io/api/app/resumes
There is a different tool that just works from the browser:
https://resumeio-to-pdf.fly.dev/
Here you need the renderingToken

Or just pay for one month and you will have updated cv's
